src/app/_services/server-config.service.ts

```diff
@@ -121,13 +121,14 @@ export class ServerConfigService {
     }
 
     const serverConfig = this.getSdkConfigServer();
-
-    if ((serverConfig.appPortal !== "") &&
-        (serverConfig.appPortal !== undefined) ) {
-          // use the specified portal
-          console.log(`Using appPortal: ${serverConfig.appPortal}`);
-          return;
-    }
+    
+    /** Commenting it out since we won't call selectPortal() if appPortal is specified in sdk-config.json */
+    // if ((serverConfig.appPortal !== "") &&
+    //     (serverConfig.appPortal !== undefined) ) {
+    //       // use the specified portal
+    //       console.log(`Using appPortal: ${serverConfig.appPortal}`);
+    //       return;
+    // }
   
```

Comment:
I believe this code should be left as is (not commented out).  Any harm in not commenting it out?  There is the entire path of when this new defaultLoadPortal API is not available (prior to 8.8.1 for now).

Reply:
No harm but we won't call this method when "appPortal" is there in sdk-config.json. I think it was being used only in that case. But, I'll uncomment.